docs/servicedns-with-externaldns.md

```diff
@@ -0,0 +1,221 @@
+# Federated Service DNS with ExternalDNS Guide
+
+Federated Service DNS (FSDNS) provides the ability to programmatically manage DNS records of Service objects.
+FSDNS is not meant to replace in-cluster DNS providers such as [CoreDNS](https://coredns.io/). Instead, FSDNS works with
+systems such as [ExternalDNS](https://github.com/kubernetes-incubator/external-dns) that are responsible for
```

Comment:
Same as #323, suggest reframing this to avoid 'such as', since external-dns is the single way we have (in this repo) to reprogram different dns provider APIs

Reply:
ack